### PR TITLE
upgrade to Node 14

### PR DESCRIPTION
**Pull request by toufali**
This PR upgrades Node from version 10 to 14 (current LTS).  This should enable package-updates to resolve vulnerabilities, and at the same time allow the latest ES features.

### CODE REVIEW

**Comment by toufali** [comment]
The issue that complicated/blocked this upgrade was an older image incompatible with Node 14.  With help from that image's code owner, @jrbenny35 , we were able to update `b4handjr/selenium-firefox:python3-latest` for Node 14 and pass the integration test.

**Review by maxxcrawford — APPROVED** [review]
This works as expected for me! I was able to do the following locally: 

- Check for breaches
- Log in
- Resolve a breach
- Add secondary email (and verify via email) 

@groovecoder When you are checking this, could you trigger a breach email (or maybe the onboarding email flow)? You may also need to check the `about:protections` page too.  

@moz-astults Can you also review this from a prod-setup standpoint? 

@pdehaan Is there any way to simulate HIBP pushing new breach data to make sure everything works as expected?

**Review by groovecoder — APPROVED** [review]
Code LGTM. I spot-checked some DB-changing things too, as I saw this updates the `pg` dependency too. Adding and removing email addresses worked, and sending a breach alert worked too. Ship it!

File: .circleci/config.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
                 command: |
                     export NVM_DIR="/opt/circleci/.nvm"
                     [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-                    nvm use 10
+                    nvm install v14.17.5 --default
```
Comment (pdehaan) [inline .circleci/config.yml:74]: Q: Should this be pinned to a specific version? Or wouldn't pinning generically to v14 use the latest (security) release in that channel? I guess my concern is that we'd have to constantly bump this if new versions of v14 are added for future security releases, etc.

Comment (toufali) [reply .circleci/config.yml]: I think you're right – updated.

**Pull request merged**